src/app/main/component/shared/components/events-list-item/events-list-item.component.ts

```diff
@@ -87,15 +99,19 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
 
   public filterTags(tags: Array<TagDto>) {
     this.itemTags.forEach((item) => (item.isActive = tags.some((name) => name.nameEn === item.nameEn)));
+
+    this.activeTags = this.itemTags.filter((val) => {
+      return val.isActive;
```

Comment:
don't use return here
this.activeTags = this.itemTags.filter((val) =>val.isActive)